docs/UserGuide.md

```diff
@@ -358,7 +360,7 @@ Duplicate handling:
 * Two resident student contacts with the same name are not allowed
 > <span style="color:Gray"> NOTE! </span> <br>
 >
-> * Constraint rationale: Phone number constraints are based on the upper and lower limit of country codes, area codes, and number digit lengths.
+> * Constraint rationale: Extremely long names would be difficult to process and display in the profile screen. 
```

Comment:
Should we also add some constraint rationales for why we only allow "-" for special characters? Though we can let Sean do that too in a different PR since he did the names and I'm not sure what we want to write.